syntaxes/styled-components.json

```diff
@@ -75,7 +75,7 @@
     },
     {
       "contentName": "source.css.scss",
-      "begin": "(?:}>|\\)\\))(`)",
+      "begin": "(?:}>)(`)",
```

Comment:
`(?:}>|\\)\\))(`)` was needed to catch multi-line rules. 
[here](https://regex101.com/r/NBlmzn/1) is an example